### ISSUE TITLE
[bug]: Comments in code preview in docs does not have enough contrast

### ISSUE THREAD

**Issue by leagrdv** [swisspost/design-system#5234]
### 🐞 Describe the bug

In the docs, whenever there is a comment in a code preview, the text is not accessible as there is not enough contrast.

### 📷 Screenshots

![Image](https://github.com/user-attachments/assets/59c15ca6-6a8d-4821-bc79-e3e596585f2d)
